ustriage/task.py

```diff
@@ -40,6 +40,7 @@ class Task:
     LONG_URL_ROOT = 'https://pad.lv/'
     SHORTLINK_ROOT = 'LP: #'
     BUG_NUMBER_LENGTH = 7
+    AGE = False
```

Comment:
`None` may be more appropriate here (it's falsy it it will work just fine with the rest of the logic).